### PR TITLE
Contributors reorg

### PR DESCRIPTION
**Pull request by karst**
This cleans up the `contributors.md` list
- Adds mber to developers list.
- Adds Spacek and Frutiemax to long term contributors.
- Moves Nubbie and wolffreak to inactive (special thanks).
~~- Orders all name-lists except for localization to be alphabetical.~~

I need some input on what else to change. I think long term contributors might need a change of name as well as its description.
~~Krutonium and I are both listed on here while we are technically developers but we don't do code reviews and whatnot instead we have different projects (audio and scenarios respectively) do we need to do something about this to keep a better overview? Perhaps move it to be part of the support team instead and a title behind it if necessary?~~ Done this

It's also a good time to check for what people don't have themselves added on this file. @frutiemax needs a place for his refactors, i don't know where that would go under, but right now he's only on the long term contributors page.

### CODE REVIEW

File: contributors.md [hunk 1]
```diff
@@ -274,28 +292,33 @@ The following people are not part of the development team, but have been contrib
 * Swedish - (Jinxit), (mharrys), (Slimeyo), Matte Andersson (Nubbie)
 
 ## Graphics
-* OpenRCT2 Logo - Yannic Geurts (xzbobzx)
-* Booster Sprites - Nicolas Hawrysh (xp4xbox)
+* Edward Calver (X7123M3-256) - Hybrid RC track & train, Single Rail RC track & train, Alpine RC track & vehicle, LIM RC extended trackset
+* Karst van Galen Last (AuraSpecs) - Steep log flume, Inverted RC Booster
+* Michael Steenbeek (Gymnasiast) - Extended fontset
+* Nicolas Hawrysh (xp4xbox) - Booster Sprites Twister, Giga, Mini, Woodie
+* Yannic Geurts (xzbobzx) - OpenRCT2 Logo
```
Comment (karst) [inline contributors.md:299]: Once #19919 is merged we also need to add all the graphics artists for this added.

File: contributors.md [hunk 2]
```diff
@@ -1,247 +1,265 @@
...
+Appreciation for long term contributors to the project having provided substantial work.
 * Edward Calver (X7123M3-256)
-* Karst van Galen Last (AuraSpecs)
-* Matte Andersson (Nubbie)
-* Kenton Boadway (Krutonium)
+* Kelson Blakewood (spacek531)
+* (frutiemax)
+
+## Special Thanks
+Appreciation for contributors who have provided substatial work, but are no longer active.
```
Comment (spacek531) [inline contributors.md:35]: substatial -> substantial

**Comment by AaronVanGeffen** [comment]
Not sure ordering everything alphabetically is an improvement tbh. Personally, I rather liked the chronological nature of the lists. I say that as someone whose name naturally at the top of alphabetically ordered lists, haha

**Comment by Broxzier** [comment]
I agree with Aaron, the chronological order is nice.

**Comment by karst** [comment]
Alright! I will change that back then :)
Is someone able to fetch the logs to see who isn't in the contributors list?

**Comment by ZehMatt** [comment]
> Alright! I will change that back then :) Is someone able to fetch the logs to see who isn't in the contributors list?

https://git-scm.com/docs/git-shortlog

**Comment by MK9999888** [comment]
Hi! Just made an account on Github for this. I helped making the sprites for diagonal brakes. Name: Martijn Kroese (MK98)

**Comment by Recurious** [comment]
Hi, together with MK98 I made the sprites for the diagonal brakes. Name: Peter van den Berg (Recurious)

File: contributors.md [hunk 3]
```diff
@@ -284,6 +284,8 @@ Appreciation for long term contributors to the project having provided substanti
 * Olivier Wervers (Oli414) - Several icons
 * Edward Calver (X7123M3-256) - Hybrid RC track & train, Single Rail RC track & train, Alpine RC track & vehicle, LIM RC extended trackset, Giga RC extended trackset
 * Karst van Galen Last (AuraSpecs) - Steep log flume, Inverted RC Booster, Giga RC powered lift
+* Martijn Kroese (MK98) - Vertical brakes project
```
Comment (spacek531) [inline contributors.md:287]: diagonal brakes?

Comment (karst) [reply contributors.md:287]: Sorry I guess I was still asleep this morning 😂

**Comment by karst** [comment]
Gonna need some help with getting that git list, it now gives me the entire list of commits when i try to execute it sorted by user rather than just a contributor list. I usually use the Desktop app, which automates most things. But that does mean I don't quite have the knowledge of git bash.

File: contributors.md [hunk 4]
```diff
@@ -52,7 +56,7 @@ The following people are not part of the development team, but have been contrib
 * Nicolas Espinosa Mooser (Nicolas-EM) - staff window refactor
 
 ## Additional implementation (OpenRCT2)
-* (atmaxinger) - User configuration
+** (atmaxinger) - User configuration
```
Comment (AaronVanGeffen) [inline contributors.md:59]: Typo?

Comment (karst) [reply contributors.md:59]: It seems thx for noticing

File: contributors.md [hunk 5]
```diff
@@ -9,6 +9,12 @@ Includes all git commit authors. Aliases are GitHub user names.
 * Tulio Paschoalin Leao (tupaschoal)
 * Olivier Wervers (Oli414)
 * Matthias Moninger (ZehMatt)
+* Karst van Galen Last (AuraSpecs) - Lead OpenSoundEffects & OpenMusic
+* Michael Bernardi (mrmbernardi)
+
+## Support team
+* Aaron van Geffen (AaronVanGeffen)
```
Comment (AaronVanGeffen) [inline contributors.md:16]: Thinking about this some more, I prefer to just be listed as a former developer. Thanks.

**Comment by karst** [comment]
I haven't been able to figure out how to pull up all the names, especially not without being thrown to death by commit keys. Someone else perhaps able to do that for me?

**Comment by tupaschoal** [comment]
I'm catching up, where do we stand here @karst , can you please summarise?

**Comment by karst** [comment]
I have done all the changes I wanted to do, except for adding all people who have provided who aren't on here. You can look at github's history for this but I personally have no idea how to pull this list. I can check it side by side no problem if I have it, but I don't know how to aquire it.

Beside that it's done.

File: contributors.md [hunk 6]
```diff
@@ -1,14 +1,19 @@
 # Contributors to OpenRCT2
-Includes all git commit authors. Aliases are GitHub user names.
+Includes all git commit authors. Aliases are GitHub usernames or Community nicknames.
```
Comment (Basssiiie) [inline contributors.md:2]: `Community` should not be capitalized.

File: contributors.md [hunk 7]
```diff
@@ -274,28 +282,36 @@ The following people are not part of the development team, but have been contrib
...
 ## RollerCoaster Tycoon 2 credits
-Design and programming by Chris Sawyer
-Graphics by Simon Foster
-Sound and music by Allister Brimble
-Additional sounds recorded by David Ellis
-Representation by Jacqui Lyons at Marjacq Ltd.
+* Allister Brimble - Music & Sound
+* Chris Sawyer - Programming & Implementation
+* David Ellis - Additional Sound
+* Jacqui Lyons (Marjacq Ltd.) - Reprsentation
```
Comment (Basssiiie) [inline contributors.md:310]: `Representation`*

File: contributors.md [hunk 8]
```diff
@@ -1,14 +1,19 @@
...
-* Hielke Morsink (Broxzier)
-* Tulio Paschoalin Leao (tupaschoal)
-* Olivier Wervers (Oli414)
-* Matthias Moninger (ZehMatt)
+* Ted John (IntelOrca) - Project Founder - Programming
+* Duncan Frost (duncanspumpkin) - Programming
+* Michał Janiszewski (janisozaur) - Programming
+* Hielke Morsink (Broxzier) - Programming
+* Tulio Paschoalin Leao (tupaschoal) - Programming
+* Olivier Wervers (Oli414) - Programming
```
Comment (Basssiiie) [inline contributors.md:10]: Despite being not so active, isn't Oli414 the lead of OpenGraphics?

**Comment by oli414** [comment]
@karst whilst you're add it can you add "Joshua Jacobs"?

**Review by tupaschoal — APPROVED** [review]

**Comment by karst** [comment]
Should be up to date again with @Gymnasiast's addition back to the team.

File: contributors.md [hunk 9]
```diff
@@ -275,15 +283,22 @@ The following people are not part of the development team, but have been contrib
...
+* Peter van den Berg (Recurious) - Diagonal brakes project
+* Joshua Jacobs - OpenGraphics Spooky Theme
 
 ## Music
-* Allister Brimble
-* Herman Riddering
 * Karst "Jalmaan" van Galen Last (AuraSpecs)
 * (Flyxxpy)
 * Jan Büchner (Greyfade)
+* Allister Brimble
```
Comment (Gymnasiast) [inline contributors.md:300]:
```suggestion
* Allister Brimble - OpenRCT2 Theme music
```

File: contributors.md [hunk 10]
```diff
@@ -275,15 +283,22 @@ The following people are not part of the development team, but have been contrib
...
+* Herman Riddering
```
Comment (Gymnasiast) [inline contributors.md:301]:
```suggestion
* Herman Riddering - Owner of the 35er Voigt, which provided new recordings for the Fairground Organ style
```

Feel free to rephrase this - my aim is just to add why he is credited.

File: contributors.md [hunk 11]
```diff
@@ -292,12 +307,14 @@ The following people are not part of the development team, but have been contrib
...
+* David Ellis - Additional Sound
+* Jacqui Lyons (Marjacq Ltd.) - Representation
+* Simon Foster - Graphics
 
 Thanks to: Peter James Adcock, Joe Booth, and John Wardley
 
-Licensed to Infogrames Interactive Inc.
+Licensed to Atari
+
+###### Copyright (c) 2014-2023 OpenRCT2 developers
```
Comment (Gymnasiast) [inline contributors.md:320]: I don’t see why this should be in the credits file.

File: contributors.md [hunk 12]
```diff
@@ -292,12 +307,14 @@ The following people are not part of the development team, but have been contrib
...
 
 ## RollerCoaster Tycoon 2 credits
-Design and programming by Chris Sawyer
-Graphics by Simon Foster
-Sound and music by Allister Brimble
-Additional sounds recorded by David Ellis
-Representation by Jacqui Lyons at Marjacq Ltd.
+* Allister Brimble - Music & Sound
+* Chris Sawyer - Programming
+* David Ellis - Additional Sound
```
Comment (Gymnasiast) [inline contributors.md:312]:
```suggestion
* David Ellis - Additional sound effects
```
AFAIK, David Ellis provided stuff like roller coaster chain recordings.

File: contributors.md [hunk 13]
```diff
@@ -292,12 +307,14 @@ The following people are not part of the development team, but have been contrib
...
+* Jacqui Lyons (Marjacq Ltd.) - Representation
+* Simon Foster - Graphics
 
 Thanks to: Peter James Adcock, Joe Booth, and John Wardley
 
-Licensed to Infogrames Interactive Inc.
+Licensed to Atari
```
Comment (Gymnasiast) [inline contributors.md:318]: Let’s revert this - the company was called Infogrames Interactive Inc. back in the day. And just "Atari" is not the legal name of the current company anyway.

**Comment by karst** [comment]
Applied your suggestions :)

File: contributors.md [hunk 14]
```diff
@@ -275,15 +283,22 @@ The following people are not part of the development team, but have been contrib
...
+* Joshua Jacobs - OpenGraphics Spooky Theme
 
 ## Music
-* Allister Brimble
-* Herman Riddering
 * Karst "Jalmaan" van Galen Last (AuraSpecs)
 * (Flyxxpy)
 * Jan Büchner (Greyfade)
+* Allister Brimble - OpenRCT2 Main Theme
+* Herman Riddering - Owner of the 35er Voigt, which provided new recordigns for the Fairground organ styles.
```
Comment (Gymnasiast) [inline contributors.md:301]:
```suggestion
* Herman Riddering - Owner of the 35er Voigt, which provided new recordings for the Fairground organ styles.
```

**Pull request merged**